library/src/main/java/com/rtchagas/pingplacepicker/ui/PlacePickerActivity.kt

```diff
@@ -114,6 +121,12 @@ class PlacePickerActivity : AppCompatActivity(), PingKoinComponent,
         // Initializes the map
         val mapFragment = supportFragmentManager.findFragmentById(R.id.map) as SupportMapFragment
         mapFragment.getMapAsync(this)
+
+        compositeDisposable.addAll(
```

Comment:
Do you know RxView.Clicks?
https://stackoverflow.com/questions/32498210/debouncing-button-clicks-using-rx

It does exactly this but in a more straightforward way. :)

I should've done that before but I forgot. Thanks for pointing this.